### ISSUE TITLE
If a URL has several audio streams, allow users to select the preffered audio stream language

### ISSUE THREAD

**Issue by ericgl** [4gray/iptvnator#390]
For example, I have a URL which streams the **Eurosport 1** TV channel.
By default, the audio stream is in **Russian** (Russian commentators).
However, there is also an audio stream in **English**. And probably, several other audio streams as well.

I have an Android app called **Televizo**, which will auto-detect the "correct" audio stream according to the selected language of the app. It doesn't provide a list to select the audio stream, but at least if I select **English** as the app's language, it will default to **English** as the audio stream. That is, of course, if a TV channel's URL provides more than one audio stream along with the video.
As a test, I switched to the Russian language, restarted **Televizo**, and when I entered the same **Eurosport** TV channel, the audio was indeed in Russian. I then switched the language back to English, restarted the app, and the audio stream went back to English.

Please expose this option in **IPTVnator** and allow us to select the audio stream whenever possible. It is very important.

**Comment by PlanK69** [comment]
yeah the fact that you can't even select the audio-langauge from a stream, is not good

**Comment by michel-cote** [comment]
I am experiencing more and more streams with "Multi" audio streams and they do not have a default set, so I cannot hear any audio.   There is no "select audio" option so I cannot select which audio stream to listen to.
